lucien-ios/LucienConstants.swift

```diff
@@ -58,4 +58,16 @@ final class LucienConstants {
 
     static let overlayButtonTopConstraint = CGFloat(10)
     static let overlayButtonLeadingConstraint = CGFloat(15)
+
+    // MARK: - Dasboard Button Constants
+
+    static let dashboardButtonCornerRadius = CGFloat(3.0)
+    static let dashboardButtonBorderWidth = CGFloat(1.0)
+    static let dashboardButtonTextSpacing = Float(1.0)
+
+    // MARK: - UIImageView Constants
+
+    static let comicCoverPhotoCornerRadius = CGFloat(6.0)
+
+
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Vertical Whitespace Violation: Limit vertical whitespace to a single empty line. Currently 2. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Avertical_whitespace)